docker-compose.yaml

```diff
@@ -0,0 +1,7 @@
+services:
+
+  nginx-ang:
```

Comment:
Bad name, not clear from the first sight. It is better to use smth sensible and clear, like `nginx-angular`

Reply:
Changed name to suggested